ons-spark/_toc.yml

```diff
@@ -51,6 +51,7 @@ parts:
   chapters:
   - file: spark-analysis/logistic-regression
   - file: spark-analysis/visualisation
+  - file: spark-analysis/Cramer_v
```

Comment:
There is a conflict can we please resolve it ?

Reply:
Changed page name to lowercase

Reply:
Will resolve before requesting review again